.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | "master" for new features / 2.7, 2.8 or 3.1 for fixes
```

Comment:
1/ It's boring to remove surrounding quotes  ;)
2/ 3.2 is now a "bug fix" branch